packages/app-admin-ui/README.md

```diff
@@ -72,14 +72,59 @@ export default {
 
 The logo to display on the signin screen.
 
-Should return a React component.
+> This must return a React component.
 
 ```javascript
 export default {
   logo: () => <MyAwesomeLogo />,
 };
 ```
 
+#### `itemHeaderActions`
+
+Header components on the Item Details page can be replaced using this hook. Ths replaces the components for item Details page for all Lists.
+
+> This must return a React component.
+
+```javascript
+// ./admin-ui/index.js
+import { ItemId, AddNewItem } '@keystonejs/admin-ui/components/'
+export default {
+  // re-implement the default AddNewItem and ItemId button + custom text
+  listHeaderActions: () => (<div><ItemId /><AddNewItem /><p>Hello world</p></div>),
+};
+```
+
+#### `listHeaderActions`
+
+Header components on the List page can be replaced using this hook. This replaces components on list page for all Lists.
+
+> This must return a React component.
+
+```javascript
+// ./admin-ui/index.js
```

Comment:
Should use the `title=./admin-ui/index.js` syntax on the line above.

Reply:
i did that but what advantage it has?

Reply:
![image](https://user-images.githubusercontent.com/3558659/83358278-1e8ea500-a3be-11ea-8ec5-8d1654cc7d4d.png)

Reply:
thought so but the rendering does not work. does leading `.` matter in file name?

Reply:
or is it thing for rendering in keystonejs.com?

Reply:
Perhaps it's the leading `.` 🤔

Reply:
I see that it is rendering on docs website